File: citame_back/src/models/user-model.js
const {Schema, model} = require('mongoose')

//Datos que se guardan en la BD
const userSchema = new Schema({

    googleId: {type: String, required: true},
    UserName: {type: String, required: true},
    EmailUser: {type: String, required: true, unique: true},
    avatar: {type: String, required: true}

});

module.exports = model('usuario',userSchema);

File: citame/lib/models/user_model.dart
class Usuario {
  final String googleId;
  final String userName;
  final String userEmail;
  final String avatar;

  Usuario(
      {required this.googleId,
      required this.userName,
      required this.userEmail,
      required this.avatar});

  factory Usuario.fromJson(Map<String, dynamic> json) {
    return Usuario(
        googleId: json['googleId'],
        userName: json['userName'],
        userEmail: json['emailUser'],
        avatar: json['avatar']);
  }
}
